src/components/Nav/Nav.js

```diff
@@ -1,7 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
+import UserForm from './UserFormComponent/userForm';
+import { userFormInfo } from './NavData';
+
+import {
+  NavContainer,
+  Navbar,
+  NavLogoBox,
+  NavLogoImg,
+  NavLeft,
+  NavMenuBox,
+  NavMenuList,
+  NavUserBox,
+  NavLogin,
+  NavSignup,
+} from './NavEle';
```

Comment:
```suggestion
import * as S from './NavEle';

// Component
<S.NavContainer />
```
만약 스타일드 컴포넌트를 별도 파일로 분리해서 사용하실거라면 위와 같이 사용하시면 됩니다.

Reply:
> 만약 스타일드 컴포넌트를 별도 파일로 분리해서 사용하실거라면 위와 같이 사용하시면 됩니다.

호오,,,, 너무 꿀팁...입니다...😃